tasks/sips/createSIP.ts

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable no-console */
+import { task } from "hardhat/config";
+import Logs from "node-logs";
+import { getAddresses, IListAddresses } from "../../configs/addresses";
+import SIPArgs, { ISipArgument } from "./args/SIPArgs";
+
+const logger = new Logs().showInConsole(true);
+     
+task("createSIP", "Create SIP to Sovryn Governance")
+.addParam("argsModuleName", "module name that is located in tasks/sips//args folder which and returning the sip arguments")
+.setAction(async ({argsModuleName}, hre) => {
+  const sipArgs: ISipArgument = await SIPArgs[argsModuleName](hre);
+  await _createSIP(hre, sipArgs)
+})
+
+export const _createSIP = async (hre, sipArgs: ISipArgument) => {
+  const { network, ethers } = hre;
+  const configAddresses: IListAddresses = getAddresses(network.name)
+  const governor = await ethers.getContractAt("IGovernorAlpha", configAddresses.governorOwner);
+
+  logger.info("=== Creating SIP ===")
+  logger.info(`Governor Address:    ${governor.address}`)
+  logger.info(`Target:              ${sipArgs.target}`)
+  logger.info(`Values:              ${sipArgs.value}`)
+  logger.info(`Signature:           ${sipArgs.signature}`)
+  logger.info(`Data:                ${sipArgs.data}`)
+  logger.info(`Description:         ${sipArgs.description}`)
+  logger.info(`============================================================='`)
+
+  await governor.propose(sipArgs.target, sipArgs.value, sipArgs.signature, sipArgs.data, sipArgs.description)
```

Comment:
it would be more reliable to log from the created proposal or to read from the proposal creation event